cmake/mission_build.cmake

```diff
@@ -40,6 +40,10 @@ function(initialize_globals)
       set(MISSION_SOURCE_DIR ${MISSION_SOURCE_DIR} CACHE PATH "Top level mission source directory")
     endif(NOT DEFINED MISSION_SOURCE_DIR)
 
+    if (NOT DEFINED CFE_SOURCE_DIR)
+      set(CFE_SOURCE_DIR "${MISSION_SOURCE_DIR}/cfe" CACHE PATH "CFE source directory")
+    endif()
+
```

Comment:
Suggestion: it may be preferable to change the name of the "project()" call above to be CFE instead of CFETOP.  Then the cmake system should defined `CFE_SOURCE_DIR` variable automatically for you.